### PR TITLE
Slack Data Models & Alert Context Helper

### PR DESCRIPTION
**Pull request by wey-chiang**
### Background

This PR adds Data Models for `Slack.AccessLogs` `Slack.AuditLogs` and `Slack.IntegrationLogs` as well as a new helper for slack alert contexts

### Testing

* Manual Testing

### CODE REVIEW

File: data_models/slack_accesslogs_data_model.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+DataModelID: Standard.Slack.AccessLogs
+DisplayName: Slack Access Logs
+Enabled: true
+Mappings:
+  - Name: source_ip
+    Path: ip
+  - Name: user_agent
+    Path: user_agent
+  - Name: actor_user
+    Path: username
```
Comment (k-bailey) [inline data_models/slack_accesslogs_data_model.yml:13]: nit: newline

File: global_helpers/panther_base_helpers.py [hunk 2]
```diff
@@ -221,6 +221,15 @@ def crowdstrike_detection_alert_context(event: dict):
     }
 
 
+def slack_alert_context(event: dict):
+    return {
+        "actor_name": deep_get(event, "actor", "user", "name", default="<MISSING_NAME>"),
+        "actor_email": deep_get(event, "actor", "user", "email", default="<MISSING_EMAIL>"),
+        "actor_ip": deep_get(event, "context", "ip_address", default="<MISSING_IP>"),
+        "user-agent": deep_get(event, "context", "ua", default="<MISSING_UA>")
```
Comment (k-bailey) [inline global_helpers/panther_base_helpers.py:229]: nit: - vs _ ? not sure if that was intentional

File: data_models/slack_integrationlogs_data_model.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+AnalysisType: datamodel
+LogTypes:
+  - Slack.IntegrationLogs
+DataModelID: Standard.Slack.IntegrationLogs
+DisplayName: Slack Integration Logs
+Enabled: true
+Mappings:
+  - Name: actor_user
+    Path: user_name
```
Comment (k-bailey) [inline data_models/slack_integrationlogs_data_model.yml:9]: not: newline

**Comment by k-bailey** [comment]
A few nit's approve the content.

**Review by k-bailey — DISMISSED** [review]

File: global_helpers/panther_base_helpers.py (hunk 2 above)
Comment (wey-chiang) [reply global_helpers/panther_base_helpers.py:229]: This was intentional -- I can change it to use `-` instead of `_`

**Review by k-bailey — APPROVED** [review]

**Pull request merged**